projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -794,6 +795,7 @@ textarea#jp-carousel-comment-form-comment-field:focus::-webkit-input-placeholder
 	margin-top: 20px;
 	color: #999;
 	position: relative;
+	overflow: hidden;
```

Comment:
Tiny nit: `overflow` is already set a few lines up. Shall we replace that one with `overflow: hidden`?

Reply:
That would make too much sense! :trollface: 

Will do. Good spotting.